project/Dependencies.scala

```diff
@@ -0,0 +1,328 @@
+import sbt.Def
+import org.portablescala.sbtplatformdeps.PlatformDepsGroupArtifactID
+import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport.*
+import sbt.librarymanagement.*
+import Versions.*
+
+object Dependencies {
+
+  private def deps(modules: PlatformDepsGroupArtifactID*)(version: String): Seq[ModuleID] =
+    modules.map(_ % version)
+
+  def In(configuration: Configuration)(dependencies: Seq[ModuleID]): Seq[ModuleID] =
+    dependencies.map(_ % configuration)
+
+    val Boopickle = Def.setting(
```

Comment:
Is this supposed to be indented this far?

Reply:
Nope, good catch.